update_usage.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+"""
+Replace usage section of README.md with current help text.
```

Comment:
because README.md is not controllable by [`sphinx autoprogram` directives](https://pythonhosted.org/sphinxcontrib-autoprogram/) etc.